doc/ITEM_CATEGORIES.md

```diff
@@ -0,0 +1,80 @@
+# Item Categories
+
+The following table lists base game item categories in their default sort order.  When adding new items, please consult this table and make sure the item gets included in the most apropriate category so it will be listed/ sorted how players will expect.
```

Comment:
```suggestion
The following table lists base game item categories in their default sort order.  When adding new items, please consult this table and make sure the item gets included in the most appropriate category so it will be listed/sorted how players will expect.
```